### ISSUE TITLE
Update for Python 3

### ISSUE THREAD

**Issue by dekaufman** [romeokienzler/developerWorks#6]
Hi,

Working through your [tutorial](https://developer.ibm.com/tutorials/iot-deep-learning-anomaly-detection-4/), it looks like IBM Cloud no longer offers Python 2 notebooks,  You might want to update the notebooks downloaded as part of the tutorial to work with Python 3.

Thanks.

**Comment by dekaufman** [comment]
Oy, now I find out how to create the Python 2 environment.  But perhaps there could be an update to the notebook to warn silly people like me that it's missing from the default Watson Studio environment set and how to create it.

**Comment by romeokienzler** [comment]
yeah, this is true (not about the silly part of course! :) you mind submitting a pull request for the updated one?
